src/watts_ui/watts_ui.py

```diff
@@ -0,0 +1,610 @@
+        app_plugin = watts.PluginPyARC(
+            template_file=plugin['template'],
+            executable=exec,
+            extra_inputs=plugin['extra_inputs'],
+            extra_template_inputs=plugin['extra_template_inputs'],
+            show_stdout=plugin['show_stdout'],
+            show_stderr=plugin['show_stderr'])
+
+    elif plugin['code'].upper() == 'RELAP5':
+        app_plugin = watts.PluginRELAP5(
+            template_file=plugin['template'],
+            executable=exec,
+            extra_inputs=plugin['extra_inputs'],
+            extra_template_inputs=plugin['extra_template_inputs'],
+            plotfl_to_csv=plugin['plotfl_to_csv'],
+            show_stdout=plugin['show_stdout'],
+            show_stderr=plugin['show_stderr'])
+
+    elif plugin['code'].upper() == 'SAS':
+        app_plugin = watts.PluginSAS(
+            template_file=plugin['template'],
+            executable=exec,
+            extra_inputs=plugin['extra_inputs'],
+            extra_template_inputs=plugin['extra_template_inputs'],
+            show_stdout=plugin['show_stdout'],
+            show_stderr=plugin['show_stderr'])
+
+        if 'conv_channel' in plugin:
+            app_plugin.conv_channel = Path(plugin['conv_channel'])
+        if 'conv_primar4' in plugin:
+            app_plugin.conv_primar4 = Path(plugin['conv_primar4'])
+
+    elif plugin['code'].upper() == 'SERPENT':
+        app_plugin = watts.PluginSerpent(
+            template_file=plugin['template'],
+            executable=exec,
+            extra_inputs=plugin['extra_inputs'],
+            extra_template_inputs=plugin['extra_template_inputs'],
+            show_stdout=plugin['show_stdout'],
+            show_stderr=plugin['show_stderr'])
+
+    elif plugin['code'].upper() == 'DAKOTA':
+        app_plugin = watts.PluginDakota(
+            template_file=plugin['template'],
+            executable=exec,
+            extra_inputs=plugin['extra_inputs'],
+            extra_template_inputs=plugin['extra_template_inputs'],
+            auto_link_files=plugin['auto_link_files'],
+            show_stdout=plugin['show_stdout'],
+            show_stderr=plugin['show_stderr'])
+
+    elif plugin['code'].upper() == 'ABCE':
+        app_plugin = watts.PluginABCE(
+            template_file=plugin['template'],
+            executable=exec,
+            extra_inputs=plugin['extra_inputs'],
+            extra_template_inputs=plugin['extra_template_inputs'],
+            show_stdout=plugin['show_stdout'],
+            show_stderr=plugin['show_stderr'])
+
+    elif plugin['code'].upper() == 'MCNP':
+        app_plugin = watts.PluginMCNP(
+            template_file=plugin['template'],
+            executable=exec,
+            extra_inputs=plugin['extra_inputs'],
+            extra_template_inputs=plugin['extra_template_inputs'],
+            show_stdout=plugin['show_stdout'],
+            show_stderr=plugin['show_stderr'])
+
+    elif plugin['code'].upper() == 'OPENMC':
```

Comment:
We need to provide documentation in WATTS developers on how they can add their plugin information to watts_ui. This needs to be done in schema and here (at minimum). I recommend providing this in `contributing.rst` file

Reply:
Updated `contributing.rst` to include instructions on how to add new plugin to `watts_ui.py` and `watts.sch`